main.py

```diff
@@ -3147,21 +3147,26 @@ async def fake(message: discord.Interaction):
 @bot.tree.command(description="(ADMIN) Force cats to appear")
 @discord.app_commands.default_permissions(manage_guild=True)
 @discord.app_commands.rename(cat_type="type")
-@discord.app_commands.describe(cat_type="select a cat type ok")
+@discord.app_commands.describe(cat_type="select a cat type ok", channel="another channel then")
 @discord.app_commands.autocomplete(cat_type=cat_type_autocomplete)
-async def forcespawn(message: discord.Interaction, cat_type: Optional[str]):
+async def forcespawn(message: discord.Interaction, cat_type: Optional[str], channel: Optional[discord.TextChannel]):
```

Comment:
summon channels can also be VoiceChannel and Thread im pretty sure

Reply:
I didn't know, sorry

Reply:
im not blaming you, just showing things to fix :3

Reply:
I should have seen more about the bot

Reply:
ok, Github didn't load your message.
thanks, Milenakos!